technologies/app/mlflow-server-1.20/mlflow-server-1.20/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM python:3.7.0
+
+RUN mkdir /mlflow/
+
+RUN apt-get update && \
+    apt-get install -y openjdk-8-jdk && \
+    apt-get clean;
+
+RUN cd / \
+    && mkdir app \
+    && cd app \
+    && wget https://archive.apache.org/dist/hadoop/common/hadoop-2.6.0/hadoop-2.6.0.tar.gz \
```

Comment:
Why not used the latest 2.6?
https://archive.apache.org/dist/hadoop/core/hadoop-2.6.5/hadoop-2.6.5.tar.gz